raiden/tests/fixtures/variables.py

```diff
@@ -21,13 +20,13 @@
 @pytest.fixture
 def settle_timeout():
     """ NettingChannel default settle timeout. """
-    return DEFAULT_SETTLE_TIMEOUT
+    return 10
```

Comment:
yes we should, you are right. Not even sure if this is a good value. Need to figure out why some other tests are failing first (could be these values).